.github/workflows/calypso-live.yml

```diff
@@ -0,0 +1,45 @@
+name: Calypso Live
+
+on:
+  pull_request:
+    types: ['opened']
+
+jobs:
+  calypso-live:
+    name: 'Launch a Calypso.live instance for your branch'
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name
```

Comment:
Do we not build calypso.live for external contributors?

Reply:
We don't in the webhook today. I based my work off that. I'm happy to open this up, although I'm not really familiar with the tool so I'll trust your call on that. Just let me know :)

Reply:
I'm not sure myself :D 

I guess it makes sense -- from a PR, you might be able make changes to the calypso.live build which would be a security concern. So we disable it for external contributors in that case. 👍 Kinda same for our other tests.

Reply:
👍 I added a comment just in case.

1493dce90fc4b50195bb08131ddf272471bd1936